apps/omg_watcher/lib/omg_watcher/exit_processor/core.ex

```diff
@@ -560,6 +564,24 @@ defmodule OMG.Watcher.ExitProcessor.Core do
     %{request | ife_input_spends_to_get: spends_to_get}
   end
 
+  @doc """
+  Filters out all the spends that have not been found (`:not_found` instead of a block)
+  This might occur if a UTXO is exited by exit finalization. A block spending such UTXO will not exist.
+  """
+  @spec handle_spent_blknum_result(list(spent_blknum_result_t()), list(Utxo.Position.t())) :: list(pos_integer())
+  def handle_spent_blknum_result(spent_blknum_result, spent_positions_to_get) do
+    {not_founds, founds} =
+      Stream.zip(spent_positions_to_get, spent_blknum_result)
+      |> Enum.split_with(fn {_utxo_pos, result} -> result == :not_found end)
+
+    {_, blknums_to_get} = Enum.unzip(founds)
+
+    warn? = !Enum.empty?(not_founds)
+    _ = if warn?, do: Logger.warn("UTXO doesn't exists but no spend registered (spent in exit?) #{inspect(not_founds)}")
```

Comment:
I propose to inline this redundant variable and use just ‚Enum.any(not_founds)’

Reply:
ah this line stirs up emotions.

I did the `warn?` to avoid a 5-line-spanning, unreadable split `mix format` proposes if I inline.

OTOH `Enum.any?([false]) != !Enum.empty?([false])` so I'm not sure if this is a viable shortcut here